### ISSUE TITLE
[Bug]: Alert is not announced to screen readers

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/vets-design-system-documentation#5783]
### DEV implementation
For the error alert to announce to screen reader users, the error text needs to be injected  into an empty element that has a `role="alert"` attribute. For reliable announcements, the wrapping element with the alert role needs to be empty on page load, then populated with the error text when the error occurs. 

### Verify the following

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.
- [ ] To my knowledge this is not an accessibility issue. If this is an accessibility issue, use the accessibility issue template

### Type of issue

Bug

### Select your product area

Digital Experience

### Team Name

CHAMPVA

### Team label

10-10d

### Point of contact

Andrea Merrill

### What happened

Error alert below continue is not announced to screen reader users, so screen reader users may not be aware of the information.

<img width="673" height="257" alt="Image" src="https://github.com/user-attachments/assets/cdce9a15-238b-4834-b7c3-9fbe243e2730" />

### What I expected to happen

Use an aria-live region to ensure this error is announced to screen reader users proactively, as in it's current location, screen reader users are unlikely to be aware of this message.

### URL where issue occurs

staging

### Steps to reproduce

Error alert not announced to screen readers

### Urgency

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [x] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

### Urgency Details

Use an aria-live region to ensure this error is announced to screen reader users proactively, as in it's current location, screen reader users are unlikely to be aware of this message.

**Comment by caw310** [comment]
@jeana-adhoc  to take a look at this as well.
